src/MOIWrapper.jl

```diff
@@ -374,3 +374,40 @@ end
 function LQOI.delete_variables!(instance::Optimizer, start_col::Int, end_col::Int)
     delete_columns(instance.inner, [Cint(i-1) for i in start_col:end_col])
 end
+
+const statmap = Dict(zip([ 0x00,  0x01, 0x02, 0x03, 0x04, 0x05],
+                     [MOI.BASIC, MOI.BASIC, MOI.NONBASIC_AT_UPPER, MOI.NONBASIC_AT_LOWER,MOI.SUPER_BASIC, MOI.NONBASIC]))
+function MOI.get(instance::Optimizer, ::MOI.ConstraintBasisStatus, i::LQOI.LCI{T}) where T <: Union{LQOI.LE, LQOI.GE, LQOI.EQ}
+    row = instance[i]
+    val = get_row_status(instance.inner, row)
+    stat = statmap[val]
+    # Single sided constraints, should not specify lower or upper
```

Comment:
Use complete English sentences in comments: http://www.juliaopt.org/JuMP.jl/v0.19.0/style/#Comments-1.